### PR TITLE
AssociationSet and Association::Name

### PR DESCRIPTION
**Pull request by solnic**

### CODE REVIEW

**Comment by solnic** [comment]
@flash-gordon no more wip, thoughts?

File: spec/support/helpers.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Helpers
+  def qualified_name(*args)
```
Comment (flash-gordon) [inline spec/support/helpers.rb]: qualified_attribute?

**Comment by flash-gordon** [comment]
@solnic 👍  looks fine to me `¯\_(ツ)_/¯`

File: spec/support/helpers.rb (hunk 1 above)
Comment (solnic) [inline spec/support/helpers.rb]: ah thanks, renamed :)

**Pull request merged**